### PR TITLE
Unpin peerDependencies

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
Unpins peer dependencies to the latest major release and updates `yarn.lock`

### CODE REVIEW

File: .changeset/fresh-avocados-divide.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@neo4j/graphql": patch
+"@neo4j/introspector": patch
```
Comment (darrellwarde) [inline .changeset/fresh-avocados-divide.md:3]: We will get away without releasing the Introspector as this was not released following the accidental pinning. 🙂 

```suggestion
```

**Review by darrellwarde — CHANGES_REQUESTED** [review]
The Introspector shouldn't be released following this. Also, is the intention to also unpin all of the _dependencies_ of the GraphQL Library and OGM? They will also be problematic.

**Comment by tbwiss** [comment]
@angrykoala what about the package tests? they also got pinned with #1914

File: packages/graphql/package.json [hunk 2]
```diff
@@ -99,7 +99,7 @@
         "semver": "7.3.7"
```
Comment (tbwiss) [inline packages/graphql/package.json:99]: question (non-blocking): What about the dependencies? They were not pinned before, https://github.com/neo4j/graphql/pull/1914/files#diff-aa225a5d16977aaf2faf11389b30a7277a5fdd8a2e36d63bf0625a337bdea6b2L90-L99

File: packages/ogm/package.json [hunk 3]
```diff
@@ -39,8 +39,8 @@
         "prettier": "2.7.1"
```
Comment (tbwiss) [inline packages/ogm/package.json:39]: question (non-blocking): Same question here, What about the dependencies?

**Review by tbwiss — COMMENTED** [review]
Just have 2 questions but I think after that it's good to go!

File: packages/graphql/package.json (hunk 2 above)
Comment (angrykoala) [reply packages/graphql/package.json:99]: Ups, good catch, forgot to push those

**Review by tbwiss — DISMISSED** [review]
LGTM now!

**Pull request merged**